### PR TITLE
Refactor tests

### PR DESCRIPTION
**Pull request by bang88**
Make tests running everywhere.

I will rebase this PR when #80 merged. or merge this PR directly.

I am not sure you have https://ci.appveyor.com/projects Account. so when you feel good i can update the appveyor icon link or edit it manually when PR merged.

### CODE REVIEW

**Comment by benjycui** [comment]
I will try AppVeyor after your PRs are merged.

**Comment by bang88** [comment]
Okay.

**Comment by benjycui** [comment]
Oops, conflict..

**Comment by bang88** [comment]
Done.

File: lerna.json [hunk 1]
```diff
@@ -1,7 +1,12 @@
 {
-  "lerna": "2.0.0-beta.36",
+  "lerna": "2.0.0-beta.38",
   "packages": [
     "packages/*"
   ],
+  "commands": {
+    "exec": {
+      "ignore": "bisheng-{theme-one,example}"
```
Comment (benjycui) [inline lerna.json]: Theme should be published.

File: package.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "scripts": {
     "lint": "eslint .eslintrc.js ./packages/*/src ./packages/*/test",
     "lint-fix": "eslint .eslintrc.js --fix ./packages/*/src ./packages/*/test",
-    "test": "lerna bootstrap && ./scripts/test.sh"
+    "test": "lerna bootstrap && lerna exec -- _mocha --recursive --opts test/mocha.opts"
```
Comment (benjycui) [inline package.json]: So, you should remove `./scripts/test.sh` at the same time?

File: lerna.json (hunk 1 above)
Comment (bang88) [inline lerna.json]: This is ignore for exec commands.

File: package.json (hunk 2 above)
Comment (bang88) [inline package.json]: I am not sure you need them or not.

Comment (benjycui) [inline package.json]: You can search to make sure if a file is useful or useless.

Comment (bang88) [inline package.json]: Hah. I will remove it. if you don't use it debugger.

**Comment by benjycui** [comment]
👍

**Pull request merged**